SongDownloadScript.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+cd SongLibrary
```

Comment:
This folder doesn't exist in the repo. Create the folder with this script inside and remove this line. Also don't end scripts with "Script", the .sh tells us that already. Naming scripts based what they do. Like any other executable.